core/modules/l2_forward.cc

```diff
@@ -96,11 +96,11 @@ static uint32_t l2_alt_index(uint32_t hash, uint32_t size_power,
 const union {
   uint64_t val[4];
   __m256d _mask;
-} _mask = {.val = {0x8000ffffFFFFfffflu, 0x8000ffffFFFFfffflu,
-                   0x8000ffffFFFFfffflu, 0x8000ffffFFFFfffflu}};
+} _mask = {.val = {0x8000ffffFFFFffffull, 0x8000ffffFFFFffffull,
+                   0x8000ffffFFFFffffull, 0x8000ffffFFFFffffull}};
 
 static inline int find_index_avx(uint64_t addr, uint64_t *table) {
-  __m256d _addr = (__m256d)_mm256_set1_epi64x(addr | ((uint64_t)1 << 63));
+  __m256d _addr = (__m256d)_mm256_set1_epi64x(addr | (1ull << 63));
```

Comment:
I know you're not making significant changes to this function (and the other static function below), but could you add a comment for these functions anyway?